deploy/install/01-prerequisite/02-serviceaccount.yaml

```diff
@@ -27,6 +27,9 @@ rules:
 - apiGroups: ["batch"]
   resources: ["jobs", "cronjobs"]
   verbs: ["*"]
+- apiGroups: ["scheduling.k8s.io"]
+  resources: ["priorityclasses"]
```

Comment:
Do we need to update `longhorn-uninstall-role` in `longhorn-manager/deploy/uninstall/uninstall.yaml`?

Reply:
I don't think so. The `Uninstall Controller` doesn't do anything to the `Priority Classes` during the uninstall process, and doesn't even monitor `Priority Classes` as part of its logic.